### PR TITLE
Add rootServiceAndSpanName function

### PR DESCRIPTION
**Pull request by tacigar**
I want to show root span information in search result list in Discover page (in #2656).
So I need a function which calculate the root span from span node tree.

I tried to implement it.
But I do not know if this implementation is correct...

WDYT? @adriancole

### CODE REVIEW

**Comment by tacigar** [comment]
> I want to show root span information in search result list in Discover page (in #2656).

Like this...

root

File: zipkin-lens/src/zipkin/trace.js [hunk 1]
```diff
@@ -266,6 +266,14 @@ function addLayoutDetails(
   }
 }
 
+export function rootServiceAndSpanName(root) {
+  const { span } = root.queueRootMostSpans().shift();
```
Comment (codefromthecrypt) [inline zipkin-lens/src/zipkin/trace.js]: I think maybe we should be less precise on a headless span, and not name the detail pane.

so, use `root.span()` and if undefined, don't name this trace.

**Comment by tacigar** [comment]
PTAL

**Comment by codefromthecrypt** [comment]
looks good. if we want to later change the headless/incomplete root span name, we have a test to change!

**Pull request merged**